src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -94,4 +97,19 @@ public File getEmbeddingDirectory() {
         return new File(getReportDirectory().getAbsolutePath(), Configuration.EMBEDDINGS_DIRECTORY);
 
     }
+
+    /**
+     * @return Patterns to be used to filter out tags in the 'Tags Overview' chart. Returns an empty list by default.
+     */
+	public Pattern[] getTagsToExcludeFromChart() {
+		return tagsToExcludeFromChart;
+	}
+
+	/**
+	 * Stores the regex patterns to be used for filtering out tags from the 'Tags Overview' chart
+	 * @param patterns Regex patterns to match against tags
+	 */
+	public void setTagsToExcludeFromChart(Pattern... patterns) {
```

Comment:
1st comment:
yes, the report will need to be rebuilt if you wanted to change the filtered tags.
the filter only affects the chart, so you still have the full information in the table below the chart.

2nd comment:
i can accept String[] but then i need to handle invalid regex.
what do you think is the best way to deal with an invalid input in this case?
maybe i can add an overload to the configuration setter that will have String[] as its input and will have a throws PatternSyntaxException in its declaration?